Automattic/wp-calypso
# Stats: Deprecate the "stats/new-all-time-highlights" feature flag

#### Proposed Changes

* Remove all usage of feature flag `stats/new-all-time-highlights`.
* Remove the legacy `StatsAllTime` component.
* Remove the legacy `StatsMostPopular ` component.

#### Testing Instructions



* Spin up this change with the Calypso Live link.
* Navigate to the Stats page `Insights`.
* Ensure everything on the page is the same as the previous one.

#### Pre-merge Checklist





Related to #70207
